modules/nf-core/estsfs/main.nf

```diff
@@ -34,7 +34,8 @@ process ESTSFS {
 
     stub:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
I am surprised this works - I always got errors when not using def while have both script and stub block.